samples/dotnet/kernel-syntax-examples/Example23_ReadOnlyMemoryStore.cs

```diff
@@ -76,12 +76,12 @@ public Task<bool> DoesCollectionExistAsync(string collectionName, CancellationTo
             throw new System.NotImplementedException();
         }
 
-        public Task<MemoryRecord?> GetAsync(string collectionName, string key, CancellationToken cancel = default)
+        public Task<MemoryRecord?> GetAsync(string collectionName, string key, bool withEmbedding = true, CancellationToken cancel = default)
```

Comment:
why is the bool true by default?

Reply:
It doesn't really make sense for this simple example to ever not return the embedding, it's just too basic. I can call that out in a comment.

Reply:
added comment explaining that the value isn't used and set = default